### DISCUSSION TITLE
Help to a newbie: How to add a network location to view the photos

### DISCUSSION THREAD

**Discussion by firstlast0** [photoprism/photoprism#1908] [Q&A] [answered]
While i was looking for a photo manager with map capability and tagging i found Photoprism.

I've installed it via Docker Windows and managed to set it up with just access to default Windows Pictures folder.

However i want to add multiple folders to view and some of them are in my basic NAS just like this:

\\192.168.1.5\NASFolder\Photos (btw the folder is not mapped, normally I'm using a desktop shortcut to access)

as well as on my computer such as:

D:\Multimedia\Photos

How can i view these different folders via Photoprism ?

Actually i have disabled the write function; I only want to be able to view, search and sort photos via the application.

First time using Docker too so any help appreciated.

Btw my yaml files related section is looking like this now:

`      HOME: "/photoprism"
working_dir: "/photoprism"
 volumes:
      ## The *originals* folder contains your original photo and video files (- "C:/Directory:/photoprism/originals"):
      - "~/Pictures:/photoprism/originals/Pictures"`

**Comment by graciousgrey** [comment, marked as answer]
Since the app is running inside a container, you have to explicitly mount the host folders you want to use. PhotoPrism won't be able to see folders that have not been mounted. See https://docs.photoprism.app/getting-started/docker-compose/#photoprismoriginals. 
An example docker-compose.yml for Windows can be found here: https://dl.photoprism.app/docker/windows/docker-compose.yml
A guide to yaml can be found here: https://docs.photoprism.app/developer-guide/technologies/yaml/

 
If you want to access "D:\Multimedia\Photos"  and "192.168.1.5\NASFolder\Photos"  instead of "~ Pictures" you need to adapt the docker-compose.yml accordingly, e.g. 

```
 volumes:
      ## The *originals* folder contains your original photo and video files (- "C:/Directory:/photoprism/originals"):
      - "D:/Multimedia/Photos:/photoprism/originals/Multimedia/Photos"
      ## Multiple folders can be made accessible by mounting them as subfolders of /photoprism/originals:
      - "//192.168.1.5/NASFolder/Photos:/photoprism/originals/NAS/Photos"
    
```
Note: Docker uses / instead of \

Note: Windows Pro users should disable the WSL 2 based engine in Docker Settings > General so that they can mount drives other than C:. See: https://docs.photoprism.app/getting-started/docker-compose/
